table/iterator_wrapper.h

```diff
@@ -80,7 +80,7 @@ class IteratorWrapperBase {
   }
   Slice key() const {
     assert(Valid());
-    return result_.key;
+    return iter_->key();
```

Comment:
I think avoiding virtual function calls to the underlying iterator is the main purpose of `IteratorWrapper`

Reply:
Addressed it.